server/repo/repo.go

```diff
@@ -136,13 +136,19 @@ func UpdateAuthRefreshToken(refreshToken string) UpdateAuthOpt {
 
 func UpdateAuthPasswordResetToken(token string) UpdateAuthOpt {
 	return func() (orm.M, error) {
-		return orm.M{orm.AuthColumns.PasswordResetToken: null.StringFrom(token)}, nil
+		return orm.M{
+			orm.AuthColumns.PasswordResetToken:           null.StringFrom(token),
+			orm.AuthColumns.PasswordResetTokenValidUntil: null.TimeFrom(time.Now().UTC().Add(24 * time.Hour)),
```

Comment:
Please extract `24 * time.Hour` to a constant and use it here and in the test. Maybe call it something like `PasswordResetTokenTTL`.

You can run `make lint` locally btw to catch linting errors prematurely.